tb/cocotb/soc/test_soc_stress.py

```
@@ -0,0 +1,265 @@
+        assert src_w == expected, (
+            f"Post-pass GPU_SRC[{i}]=0x{src_w:08x} != expected 0x{expected:08x} "
+            f"(initial=0x{i+0x11:02x} + STRESS_ITER={STRESS_ITER})"
+        )
+        assert dst_w == expected, (
+            f"Post-pass GPU_DST[{i}]=0x{dst_w:08x} != expected 0x{expected:08x} "
+            f"(GPU_SRC_UPDATE should have set DST==SRC after last GPU_VERIFY)"
+        )
+
+    # ── Verify timer is non-zero (proves AXI-Lite timer path was reached) ─────
+    assert timer_snap != 0, (
+        f"Timer snapshot is 0 — timer AXI-Lite read path not exercised or "
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)